dialogue-core/src/main/java/com/palantir/dialogue/core/RttSampler.java

```diff
@@ -104,7 +104,7 @@
             OptionalLong rtt = snapshots[i];
             float rttSpectrum = rtt.isPresent() ? ((float) (rtt.getAsLong() - bestRttNanos)) / rttRange : 0;
             if (rttSpectrum < 0f || rttSpectrum > 1f) {
-                log.warn(
+                log.info(
```

Comment:
possibly even down to `debug` since we're confident it wasn't wildly over here?

Reply:
and I guess an `isDebugEnabled` maybe?

Reply:
I wanted to leave it at a level that's default enabled for rollout so we can validate against outliers just in case. Perhaps in a week we could reduce it.